src/VisualStudio/Core/Def/InheritanceMargin/MarginGlyph/InheritanceMarginContextMenu.xaml.cs

```diff
@@ -25,20 +26,31 @@ internal partial class InheritanceMarginContextMenu : ContextMenu
     private readonly IUIThreadOperationExecutor _operationExecutor;
     private readonly Workspace _workspace;
     private readonly IAsynchronousOperationListener _listener;
+    private readonly double _scaleFactor;
 
     public InheritanceMarginContextMenu(
         IThreadingContext threadingContext,
         IStreamingFindUsagesPresenter streamingFindUsagesPresenter,
         IUIThreadOperationExecutor operationExecutor,
         Workspace workspace,
-        IAsynchronousOperationListener listener)
+        IAsynchronousOperationListener listener,
+        double scaleFactor)
     {
         _threadingContext = threadingContext;
         _streamingFindUsagesPresenter = streamingFindUsagesPresenter;
         _workspace = workspace;
         _operationExecutor = operationExecutor;
         _listener = listener;
+        _scaleFactor = scaleFactor;
         InitializeComponent();
+        SetZoom();
+    }
+
+    private void SetZoom()
+    {
+        var zoomTransform = new ScaleTransform(_scaleFactor, _scaleFactor);
+        zoomTransform.Freeze();
+        LayoutTransform = zoomTransform;
```

Comment:
nit.  can this be a local function?  or just inline entirely in to the constructor.